modules/admin_manual/pages/configuration/server/security/hsmdaemon.adoc

```diff
@@ -14,12 +14,12 @@
 :softhsm2-url: https://www.opendnssec.org/softhsm/
 :unix-sockets-url: http://beej.us/guide/bgipc/html/multi/unixsock.html
 
-== Introduction
-
-The hsmdaemon is a daemon to delegate encryption to an {hsm-url}[HSM (Hardware Security Module)].
-It can be necessary, as PHP cannot directly interface with {pkcs11-url}[a PKCS11 stack]; neither by an API wrapper, because one does not exist, yet, nor via the OpenSSL bindings
+The hsmdaemon is a daemon, provided by ownCloud, to delegate encryption to an {hsm-url}[HSM (Hardware Security Module)].
+This can be necessary, as PHP cannot, directly, interface with {pkcs11-url}[a PKCS11 stack]; neither with an API wrapper, because one does not exist, nor via the OpenSSL bindings.
 Because of this, a separate process is needed to decrypt anything with the private key stored in an HSM. 
 
+NOTE: The encryption app, which is bundled with ownCloud Server, will have out-of-the-box support for the hsmdaemon with the upcoming 10.2 release.
```

Comment:
Would´nt it be better to write:
`for the hsmdaemon with the upcoming 10.2 release` --> `for the hsmdaemon with release 10.2 upwards`.
Because at one point in time upcoming will be present respectively past...

Reply:
It will. The text can be updated when 10.2's released to remove the specific reference to that version.

Reply:
I think you can just remove the reference as the productizing of this whole feature will anyway take place with/after 10.2. Maybe just state that you need Server >= 10.2 for HSM support.